game/templates/game/leaderboard.html

```
@@ -69,5 +69,39 @@ <h1>🏆 Puzzle Leaderboard</h1>
+    <table>
+        <thead>
+            <tr>
+                <th>Rank</th>
+                <th>User</th>
+                <th>Rating</th>
+                <th>Games Played</th>
+                <th>Wins</th>
+                <th>Loss</th>
+                <th>Draws</th>
+            </tr>
+        </thead>>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)